lib/VhostContainer.php

```diff
@@ -10,9 +10,11 @@ class VhostContainer implements \Countable, Monitor {
     private $defaultHttpDriver;
     private $setupHttpDrivers = [];
     private $setupArgs;
+    private $options;
 
-    public function __construct(HttpDriver $driver) {
+    public function __construct(HttpDriver $driver, Options $options = null) {
         $this->defaultHttpDriver = $driver;
+        $this->options = $options;
```

Comment:
You ought using `$ireq->client->options` instead of dependency injecting it here.

Reply:
Sure, simple enough.